.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## Description of changes
+What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.
```

Comment:
Could you also just mention that a description of how to test can be included here as well.

Reply:
Yup.  Added in eb8b236e29